src/main/java/at/porscheinformatik/sonarqube/licensecheck/gradle/GradleDependencyScanner.java

```diff
@@ -40,13 +43,12 @@ public GradleDependencyScanner(LicenseMappingService licenseMappingService)
     @Override
     public Set<Dependency> scan(SensorContext context)
     {
-        File moduleDir = context.fileSystem().baseDir();
-
         Map<Pattern, String> defaultLicenseMap = licenseMappingService.getLicenseMap();
 
-        File licenseDetailsJsonFile = new File(moduleDir, "build" + File.separator + "reports" + File.separator
-            + "dependency-license" + File.separator + "license-details.json");
-
+        String pathDef = context.config().get(JSON_REPORT_PATH_PROPERTY).orElse(JSON_REPORT_PATH_DEFAULT);
+        LOGGER.debug("Searching for license file at {}", pathDef);
+        File licenseDetailsJsonFile = context.fileSystem().workDir().toPath().resolve(pathDef).toFile().getAbsoluteFile();
+        LOGGER.debug("Resolved path: {}", licenseDetailsJsonFile.getPath());
```

Comment:
The two debug logs are redundant - please remove one (you might add the info if the file was actually found)